src/main/java/sum/ExecutorServiceSum.java

```diff
@@ -0,0 +1,36 @@
+package sum;
+
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.stream.Collectors;
+import org.apache.commons.collections4.ListUtils;
+
+public class ExecutorServiceSum {
+    public static final int THREADS = 10;
+    private final List<Integer> digits;
+
+    public ExecutorServiceSum(List<Integer> digits) {
+        this.digits = digits;
+    }
+
+    public Integer calculate() {
+        ExecutorService executorService = Executors.newFixedThreadPool(THREADS);
+        List<List<Integer>> partThread = ListUtils.partition(digits, digits.size() / THREADS);
+        List<MyCallable> myCallables = partThread.stream()
```

Comment:
```suggestion
        List<Callable> myCallables = partThread.stream()
```
it's a good practice to use abstraction instead

Reply:
Fixed